integration-tests/integration/features/matching.feature

```diff
@@ -83,6 +83,47 @@ Feature: Matching defendants to nDelius records
     Then I should be on the "Defendants with possible nDelius records" page
     And I should see the no record match confirmation banner message
 
+  Scenario: Manually match a defendant and submit without entering a CRN
+    Given I am an authenticated user
+    When I navigate to the "/match/bulk/" route for today
+    And I am using the "bulkList" match data
+    Then I should be on the "Defendants with possible nDelius records" page
+    When I click the "Review records" link
+    Then I should be on the "Review possible nDelius records" page
+    And I am using the "defendantOneRecords" match data
+    And I click the element with id "defendant-1"
+    And I click the "Can't see the correct record?" summary link
+    And I click the "link it to them with a case reference number" link
+    Then I should be on the "Link an nDelius record to the defendant" page
+    And I should see the body text "Use a case reference number (CRN) to link to an existing nDelius record to the defendant."
+    And I should not see the heading level 2 with text "EDefendant details"
```

Comment:
Think this must be a typo and it should be checking that "Defendant details" is there rather than "EDefendant details" isn't. Will amend